forms/CommentCreateForm.php

```diff
@@ -67,9 +70,11 @@ public function save()
         $Comment = $this->Comment;
 
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::$app->getModule(Comments\Module::NAME)->model('comment');
```

Comment:
I understood. I think to remove the method `model` from the module. Since this is misleading.